assets/src/edit-story/types.js

```diff
@@ -146,8 +153,8 @@ StoryPropTypes.elements.text = PropTypes.shape({
 });
 
 StoryPropTypes.elements.shape = PropTypes.shape({
-  ...StoryElementPropsTypes,
-  backgroundColor: PropTypes.string,
+  ...StoryElementPropTypes,
+  resource: StoryPropTypes.colorResource,
```

Comment:
I'm not sure I understand this. What resource does this represent?

Reply:
Maybe the name 'resource' isn't super great here but in essence, a resource is a group of core attributes that define the particular instance/type of the element as opposed to generic properties like x/y/width/height/etc. Resources can be passed around to transform one element to another without the need to carry all other attributes @dvoytenko  what do you think?

Reply:
TBH, in this form, the color resource specifically doesn't make much sense to me. I.e. that's just a background color for now, right? Eventually we might have a case where we have external SVGs used for brushes and that's where we may need resources. But we don't have that yet.